doc/docs/master/deploy-manage/deploy/deploy-pachyderm-ide.md

```diff
@@ -0,0 +1,103 @@
+# Deploy the Pachyderm IDE
+
+!!! note
+    The Pachyderm IDE is an enterprise feature,
+    which is also available for testing during
+    the 14-day free trial.
+    Contact sales@pachyderm.com for more
+    information.
+
+The Pachyderm Integrated Development Environment (IDE) is
+an optional extension to Pachyderm clusters that provides a
+comprehensive environment for prototyping and deploying
+Pachyderm pipelines, as well as introspecting on data in the
+Pachyderm version control system. It combines together Jupyter,
+JupyterHub, and JupyterLab, which are familiar tools for many data
+scientists.
+
+JupyterLab is the newest Jupyter web interface that
+allows you to run your Jupyter notebooks in tabs and extend the
+tabs with custom applications. Our version of JupyterLab comes
+with `pachctl`, `python-pachyderm`, Pachyderm shell,
+Terminal, Python 3 Notebooks, and other useful components preconfigured.
+From the terminal window, you can use `pachctl` commands and from 
+within the notebooks you can call 
+[Pachyderm Python client library](https://github.com/pachyderm/python-pachyderm)
+methods to manage Pachyderm directly from the
+JupyterLab UI.
+
+The following diagram shows the Pachyderm and JupyterHub deployment.
+
+![JupyterHub and Pachyderm Architecture Overview](../../assets/images/d_jupyterhub-pachyderm-arch.svg)
+
+In the diagram above, you can see that Pachyderm and JupyterHub are
+deployed on the same Kubernetes cluster. You deploy Pachyderm by
+using the pachctl `deploy command` and JupyterHub by running
+our deployment script as described below. After deployment, you log in
+to JupyterHub with your Pachyderm user and interact with Pachyderm
+from within the Jupyter UI by using the Pachyderm Python client.
```

Comment:
pachyderm-python is the primary, but not only way of interacting with pachyderm nowadays. Not sure if that should be clarified or not.

Reply:
added